app/Models/SettingsProperty.php

```diff
@@ -68,5 +68,22 @@ public function registerMediaConversions(Media $media = null): void
             ->performOnCollections('logo', 'dark_logo')
             ->height(250)
             ->keepOriginalImageFormat();
+
+        $this->addMediaCollection('portal_favicon')
+            ->singleFile()
+            ->acceptsMimeTypes([
+                'image/png',
+                'image/jpeg',
+                'image/ico',
+                'image/webp',
+                'image/jpg',
+                'image/svg',
+            ]);
```

Comment:
This is not the right method to define Media Collections in. I believe you are supposed to do so in a `registerMediaCollections` method.

https://spatie.be/docs/laravel-medialibrary/v11/working-with-media-collections/defining-media-collections

Reply:
Done. sorry again for silly mistakes.

Reply:
No problem!